charts/posit-chronicle/ci/complex-values.yaml

```diff
@@ -0,0 +1,65 @@
+image:
+  tag: "latest"
+  imagePullPolicy: "IfNotPresent"
+
+serviceaccount:
+  create: true
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::123123123123123:role/chronicle-serviceaccount-role
+
+service:
+  # -- The port to use for the REST service
+  port: 12345
+  annotations:
+    test-annotation: test-service-annotation
+  labels:
+    test-label: test-service-label
+    test-label2: test-service-label2
+  selectorLabels:
+    test-selector-label: test-service-selectorLabel
+    test-selector-label2: test-service-selectorLabel2
+
+pod:
+  replicas: 4
+  command: "/bash"
+  args: ["echo", "hello world"]
+  # -- Optional environment variables
+  env:
+  - name: "TEST_ENV_VAR"
+    value: "test-value"
+  annotations:
+    test-annotation: test-service-annotation
+  labels:
+    test-label: test-service-label
+    test-label2: test-service-label2
+  selectorLabels:
+    test-selector-label: test-service-selectorLabel
+    test-selector-label2: test-service-selectorLabel2
+
+storage:
+  persistentVolumeSize: 10Gi
+
+config:
```

Comment:
Definitely a worthwhile callout. We've done a lot of work already to pare down this config file, and went even further within this PR migrating from https://github.com/rstudio/platform-helm/blob/main/charts/chronicle/values.yaml, so this will likely continue as we get more comfortable with our default settings